Umbraco-Cloud/Set-Up/Working-Locally/index.md

```diff
@@ -189,7 +189,7 @@ dotnet sln add .\src\MyAwesomeProject.Web\MyAwesomeProject.Web.csproj
 
    ![Class library](images/class-library.png)
 
-Once the Class library has been added, you can see the project(s) that have been added in Solution Explorer:
+Once the Class library has been added (in this case named it".Core"), you can see the project(s) that have been added in Solution Explorer:
```

Comment:
@HamDerAndrew  The wording seems a bit odd to me 🤔

Should it maybe be "In this case it is named .Core"?